SmartSaver.WPF/MainWindow.xaml.cs

```diff
@@ -150,5 +134,32 @@ private void ListView_SelectionChanged(object sender, SelectionChangedEventArgs
             _user.Account.GoalEndDate = (DateTime)goalDateBox.SelectedDate;
             _context.SaveChanges(); // Eilute įrašo į duomenų bazę.
         }
+
+        private void UpdateHistoryTable()
+        {
+            List<Transaction> transactions =
+                _context.Transactions.Where(t => t.AccountId.Equals(_user.Account.Id)).ToList();
+
+            HistoryTable.ItemsSource = transactions;
+        }
+
+        private void EnableTabs()
+        {
+            statusTab.IsSelected = true;
+            statusTab.IsEnabled = true;
+            historyTab.IsEnabled = true;
+            savingPlansTab.IsEnabled = true;
+            entriesTab.IsEnabled = true;
+            accountTab.IsEnabled = true;
+            logInTab.IsEnabled = false;
+        }
+
+        private void PrepareData()
+        {
+            categoryBox.ItemsSource = _context.Categories.ToList().Select(s => s.Title);
```

Comment:
as I see Id = 1 is default for expenses, so where clause is needed here to not add that category to a categoryBox